### PR TITLE
still cannot get MVP to display new Input

### PR DESCRIPTION
**Pull request by Sem8**

### CODE REVIEW

File: instagram/src/components/CommentSection/CommentSection.js [hunk 1]
```diff
@@ -1,21 +1,63 @@
...
+    }
+    
+    handleChanges = (e) => {
+        this.setState({ newComment: e.target.value })
+    };
+    
+    handleAddNewComment = (e) => {
+        e.preventDefault();
+        this.setState({ 
+            allComments: [...this.state.allComments, { username: 'Sem', text: this.state.newComment}],
```
Comment (Mindysueasaurus) [inline instagram/src/components/CommentSection/CommentSection.js:39]: This is very nice and organized. Great job on this method.

**Comment by Sem8** [comment]
MVP is somewhat working but and page can display upon log in. But Login button and input fields disappear on log in and don't reappear even if I refresh the page. Only way to bring back Log in button is if I go to Application tab in dev tools and clear local storage. Was this bug supposed to be part of our MVP?

File: instagram/src/authentication/authenticate.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        componentDidMount() {
+            if(!localStorage.getItem('user')) {
+                this.setState({loggedIn: false});
+            } else {
+                this.setState({ loggedIn: true });
+            }
+        }
+
+        render() {
+            if(this.state.loggedIn == true) {
```
Comment (Mindysueasaurus) [inline instagram/src/authentication/authenticate.js:22]: Great job here.

**Review by Mindysueasaurus — COMMENTED** [review]
Greta job Sem. I would suggest going through after today and cleaning up your files. Removing everything that isn't an actual comment and any console.logs you were using to check your data.

**Pull request merged**